_blocks/develop.md

```diff
@@ -2,7 +2,7 @@
 lang: nl
 index: 2
 title: Ontwikkel mee
-subtitle: De app is open source en je kunt de ontwikkeling volgen op Github.
```

Comment:
This is not a minor change :wink:, this would deviate from the `open source` mentioned in [the timeline (Dutch)](https://www.rijksoverheid.nl/onderwerpen/coronavirus-app/tijdpad-proces-coronavirus-app).

@EdoPlantinga any changes on the open/public source definition of the app?